### PR TITLE
generate changelog from github

### PR DESCRIPTION
**Pull request by ashleywang1**
BOT NOTES: 
resolves https://github.com/solo-io/gloo/issues/2799

### CODE REVIEW

File: docs/cmd/generate_changelog_doc.go [hunk 1]
```diff
@@ -70,6 +75,19 @@ func changelogMdCmd(opts *options) *cobra.Command {
...
 }
 
+func changelogMdFromGithubCmd(opts *options) *cobra.Command {
+	app := &cobra.Command{
+		Use:   "gen-changelog-md-from-github",
+		Short: "generate a markdown file from Github Release pages API",
+		RunE: func(cmd *cobra.Command, args []string) error {
+
+			return generateChangelogMdFromGithub(opts, args)
+			return nil
```
Comment (kdorosh) [inline docs/cmd/generate_changelog_doc.go:85]: unreachable code

File: docs/cmd/generate_changelog_doc.go [hunk 2]
```diff
@@ -167,3 +185,44 @@ func generateChangelogMd(opts *options, args []string) error {
...
+			Page:    0,
+			PerPage: 10000000,
+		})
+	if err != nil {
+		return err
+	}
+
+	sort.Slice(allReleases, func(i, j int) bool {
+		return *allReleases[i].TagName > *allReleases[j].TagName
+	})
```
Comment (kdorosh) [inline docs/cmd/generate_changelog_doc.go:221]: can we sort chronologically instead?

Comment (ashleywang1) [reply docs/cmd/generate_changelog_doc.go:221]: ah, sure! The releases are automatically returned in chronological order.

**Comment by ashleywang1** [comment]
/kick untar-mod-cache error? This seems to be popping up a lot more often recently and its weird.

**Comment by ashleywang1** [comment]
/kick gateway clean up the envoy config when the virtual service is deleted timing flake

**Pull request merged**